src/Header.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Header from './Header.js';
+import renderer from 'react-test-renderer';
+
+it(' Header.js renders correctly', () => {
+  const tree = renderer
+    .create(< Header.js />)
```

Comment:
so the reason why your tests are failing is because you have that js inside of the class component, just remove that js and they should start passing!